packages/animations/lib/src/dual_transition_builder.dart

```diff
@@ -0,0 +1,228 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
```

Comment:
Might be a small detail, but does the year on this matter? I know the `flutter/flutter` project consolidated the years to be the same year a few months back.

Reply:
We don't have rules established for this repository yet, so using the current year should be fine until we decide to unify.